doc/error_case.puml

```diff
@@ -0,0 +1,40 @@
+@startuml
+actor User
+participant Powermeter
+participant Error_Manager
+participant EVSE_Manager
+participant OCPP
+
+User -> EVSE_Manager: startChargingSession
+alt successful case
+EVSE_Manager -> Powermeter: startTransaction
+Powermeter -> EVSE_Manager: startTransaction Response (OK/ID)
+EVSE_Manager -> OCPP: persistTransaction (ID)
+else startTransaction failing due to power loss
+EVSE_Manager -> Powermeter: startTransaction
+Powermeter -> EVSE_Manager: startTransaction Response (FAIL)
+EVSE_Manager -> User: stopChargingSession
```

Comment:
really nice that you've created these puml files, maybe we should come up with a guideline, whether rendered versions (jpeg, png) should be created - because it is quite some work to manually create a picture out of it